bibliopixel/image.py

```diff
@@ -75,7 +75,7 @@ def loadImage(led, imagePath="", imageObj=None, offset=(0, 0), bgcolor=colors.Of
 
     def setter(x, y, pixel):
         if y >= 0 and x >= 0:
-            texture[y][x] = (r, g, b)
+            texture[y][x] = pixel
```

Comment:
I didn't do this!!!  I mean, I did create this file but I moved that code from elsewhere!!   Honest!!!